week2/homework/src/index.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const commandList = require('./commandsList');
+const theMethods = require('./methods.js');
```

Comment:
Not an "error", but generally variable names do not contain articles ("the", "a")

Reply:
Just "methods" would look great!

Reply:
done.